### ISSUE TITLE
Offset display

### ISSUE THREAD

**Issue by JunCEEE** [European-XFEL/EXtra-geom#103]
I wondered if the offset in #101 is considered for displaying the geometry (`geom.inspect()`) since I didn't see a big difference between the plots of huge offsets (upper) and small offsets (lower).

![image](https://user-images.githubusercontent.com/3163505/131099089-c46fa89f-9196-4382-bcc2-0e3969aa7e73.png)

![image](https://user-images.githubusercontent.com/3163505/131099035-a7157fcb-2ba7-480f-bdfa-abe1e2d2e548.png)

**Comment by takluyver** [comment]
No, inspect ignores the offset, since typically the detector is very close to a flat plane, and presenting a third dimension is more complex. Ideas are welcome for how we could let people see that.
